src/content/subgames.cpp

```diff
@@ -350,6 +350,7 @@ void loadGameConfAndInitWorld(const std::string &path, const std::string &name,
 		Settings conf;
 
 		conf.set("world_name", name);
+		conf.set("world_uuid", generate_uuid4());
```

Comment:
I don't have a good argument as to why, but I'd prefer if the UUID was only generated if announcements are enabled.

Reply:
I added a commit to do this.